src/engine/Map.cxx

```diff
@@ -819,7 +819,7 @@ bool Map::setTileID(const std::string &tileID, Point coordinate)
       currentMapNode.getSprite()->setRenderFlag(layer, true);
     }
 
-    if (!targetCoordinates.size() == 1)
+    if (!(targetCoordinates.size() == 1))
```

Comment:
don't change this. i know the code is wrong, but it doesn't work if you change it. I'm currently rewriting setTileID (again) to fix this, but this change will break placement